src/vue/modules/coinpayments-deposit/components/pending-issuances-table.vue

```diff
@@ -7,6 +7,9 @@
       :asset="item.asset"
       :address="item.address"
       :key="index"
+      :selected-issuance="selectedIssuance"
+      @issuance-selected="issuanceSelected"
+      @reset-issuance-selection="resetIssuanceSelection"
```

Comment:
1. why does an entry of a list should know anything about other entries? remove that `selected-issuance` prop completely
2. why does an entry of a list should command other entries to do anything? remove that `reset-issuance-selection` prop also
3. `issuance-selected` should be renamed to something like `row-expanded` or `entry-expanded`

the logic of contracting of an entry should be handled by the list, not by the entries. the maximum responsibility of the entry is to warn the list that it has been expanded